package.json

```diff
@@ -397,6 +402,15 @@
 					"light": "images/title/light/icon-issue.svg"
 				}
 			},
+			{
```

Comment:
@mohitsuman please let me know where we can keep this survey button and suggest any opt icon as well